SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -112,6 +112,16 @@ void ASpatialFunctionalTest::RegisterAutoDestroyActor(AActor* ActorToAutoDestroy
 	}
 }
 
+void ASpatialFunctionalTest::LogStep(ELogVerbosity::Type Verbosity, const FString& Message)
+{
+	Super::LogStep(Verbosity, Message);
+
+	if(Verbosity == ELogVerbosity::Error || Verbosity == ELogVerbosity::Fatal)
+	{
+		FinishTest(EFunctionalTestResult::Failed, TEXT("Failed assertions"));
```

Comment:
Maybe we should also set the current step index to something invalid - or is that already handled on FinishTest?

Reply:
That's handled by the Test Authority as a flow to do the cleanup process as well.